static/js/components/dashboard/courses/StatusMessages.js

```diff
@@ -88,6 +88,31 @@ const messageForNotAttemptedExam = (course: Course) => {
   return message
 }
 
+const messageForAttemptedExams = (course: Course, passedExam: boolean) => {
+  const whenFailed = passedExam ? "" : "You did not pass the exam. "
+  const payAgain = course.has_to_pay
+    ? "If you want to re-take the exam, you need to pay again. "
+    : ""
+  let whenToSchedule = ""
+  if (course.can_schedule_exam) {
+    // if can schedule exam now
+    if (!course.has_to_pay) {
+      whenToSchedule = "Click above to reschedule an exam with Pearson."
+    }
+  } else if (R.isEmpty(course.exams_schedulable_in_future)) {
```

Comment:
use loadashhttps://lodash.com/docs/4.17.10#isEmpty because it tasked null and '' as empty.

Reply:
I think `exams_schedulable_in_future` should always be "" when no exams, but i'll change this just in case.

Reply:
Sorry, I confused this with can_schedule_exam. This exams_schedulable_in_future is a list so what is wrong with using R.isEmpty?

Reply:
R.isEmpty is ok for list